dashboard design based on the dataset description has been devised

salva-u opened the pull request:
Please have a look at the dashboard design, I can breakdown the columns that were used and/or added here, which if everyone agrees is okay, we can reflect in the data part?
Original columns kept: gender, age_range, race, higher_education, political_party, ideology, trump_approval, fairness_voting, america_survival_100, trump_nuclear,
Columns added:
`war_likelihood`: Based on suggestion under dataset PR. Yes = 1 and No = 0, a score from 0 to 2 whether america will go to war with russia and china
`monetary_anxiety_score`: No Anxiety = 0 , High anxiety = 5, Medium Level Anxiety = 2

Columna to be ignored: 
monetary_anxiety, preferential_voting,woman_president_25, universal_healthcare_25, humanity_survival_100, trump_2020 ,america_vs_russia_50, america_vs_china_50, better_world, political_spectrum_score, time_answered,

Delete these columns after adding the feature engineered columns:
america_vs_russia_50, america_vs_china_50,

zth96 commented:
<img width="1088" alt="image" src="https://github.com/UBC-MDS/DSCI-532_2024_14_mds_saves_america/assets/143483779/302703a5-bccc-4eda-a30b-3d04a20eba1a">

Thank you for drafting the dashboard design! 

Here are couple of suggestions from my aspect

1, For the bottom right graph fairness of voting vs approval of trump, the x and y axis would be more interpretable if they are in a scale order. e.g. having 'strongly disapprove" to 'strongly approve' and likewise from 'very confident' to not confident'. And may be consider a different color for the title? Grey can be hard to read on this background

2, Would it be better to have age range and education level as slider instead of dropdown menu filter? I feel like slider can support more flexible filter requests. 

3, Maybe we need to add a data source citation on the dashboard? (I'm not 100% sure)

4, Just suggestive, can we build some side by side comparative features? I feel like for current design we can only see the result for one demographic group and switch to other on the top filters. Is it workable to have two graphs for us to directly do visual comparison?

Sampsonyu commented:
Thanks for the sketch. It looks sufficient for Milestone 1, with room for future adjustments. 

Could you confirm if the filter pane utilizes dropdown menus? It'd be helpful to specify this in the description. Additionally, considering that three out of four charts are bar charts, should we explore different chart types? For instance, we had mentioned a donut chart during our lab discussion.

salva-u commented:
> Could you confirm if the filter pane utilizes dropdown menus? It'd be helpful to specify this in the description. 

Good point, added.

>Additionally, considering that three out of four charts are bar charts, should we explore different chart types? For instance, we had mentioned a donut chart during our lab discussion.

Yeah I removed it because it was taking up more space, added it back for one of the charts.

@Sampsonyu

salva-u commented:
> Here are couple of suggestions from my aspect

> 1, For the bottom right graph fairness of voting vs approval of trump, the x and y axis would be more interpretable if they are in a scale order. e.g. having 'strongly disapprove" to 'strongly approve' and likewise from 'very confident' to not confident'. And may be consider a different color for the title? Grey can be hard to read on this background

Yeah, it would be a pain to change it now, but I think it a rough sketch, should I add a comment about it being ordered. Or we can order it ourselves? Yes, you're right about the font colors , I made it almost black and change the color of the title. I think we can play around with the font colors later too?

> 2, Would it be better to have age range and education level as slider instead of dropdown menu filter? I feel like slider can support more flexible filter requests.

Okay sure, I think we can leave it slightly open ended I will specify for now drop downs and sliders. We can add sliders/drop downs where it makes sense? Mentioned slider for age range in the side description.

> 3, Maybe we need to add a data source citation on the dashboard? (I'm not 100% sure)

I think we can do without it? since the proposal and repo readme will mention data source, I am not sure, I can add it if you guys think we should have it?

> 4, Just suggestive, can we build some side by side comparative features? I feel like for current design we can only see the result for one demographic group and switch to other on the top filters. Is it workable to have two graphs for us to directly do visual comparison?

That is a good point, Initially I wanted to keep a small section with demographic comparisons, but then we won't really need the sliders, because it is kind of useless, that is why I changed it, I am not sure, I can add them back but the need for the sliders will be redundant, since we wanna compare them? Also I did keep racial group and age range charts with their breakdowns so we still had some demographic comparison, do you think that would suffice? or we need to change it more?

@zth96

zth96 reviewed (approved):
Looks great!
